Simplenote/SPLegacyCredentials.swift

```diff
@@ -101,13 +105,13 @@ class SPCredentials: NSObject {
         }
     }
     
-    @objc static func WPCCClientSecret() -> String {
-        if let value = configDictionary().value(forKey: "WPCCClientSecret") {
-            return (value as! String)
-        } else {
-            return ""
-        }
-    }
+//    @objc static func WPCCClientSecret() -> String {
```

Comment:
Do we want to keep this around if it's commented out? Would it be better to just remove it entirely?

Reply:
Update: This file is gone! thank you!